applications/luci-app-unbound/luasrc/model/cbi/unbound.lua

```diff
@@ -179,11 +194,17 @@ rsc.rmempty = false
 ag2 = s1:taboption("resource", Value, "root_age", translate("Root DSKEY Age:"),
   translate("Limit days between RFC5011 to reduce flash writes"))
 ag2.datatype = "and(uinteger,min(1),max(99))"
-ag2:value("14", "14")
-ag2:value("28", "28 ("..translate("default")..")")
-ag2:value("45", "45")
-ag2:value("90", "90")
+ag2:value("3", "3")
+ag2:value("9", "9 ("..translate("default")..")")
+ag2:value("12", "12")
+ag2:value("24", "24")
 ag2:value("99", "99 ("..translate("never")..")")
 
+tgr = s1:taboption("resource", Value, "trigger", translate("Trigger Networks:"),
```

Comment:
Without this unbound does not reload when an (LAN) interface goes up or down ?
If not so, would a checkbox to trigger on all interfaces except those listed be an idea (and then, by default, include wan6 that supposedly triggers to often (?) in the skiplist)